### ISSUE TITLE
Failing to start dev branch: sed: ./dist/index.html: No such file or directory

### ISSUE THREAD

**Issue by bakstad** [pantsel/konga#24]
when starting the dev branch I get this error: 

sed: ./dist/index.html: No such file or directory

The Dockerfile does not seem to properly build the frontend

**Comment by pantsel** [comment]
You need to run cd frontend && gulp dist before building.

That way only the static frontend files will be used in the image thus saving 100mb+ of compressed resources.

There is a comment about that in <code>start.sh</code>

**Comment by bakstad** [comment]
That is not a good solution. The Dockerfile should be able to built it self entirely. 

To save space you could clean up all the uncompressed resources at the end of the dockerfile.

With the current setup automated builds are impossible.

**Comment by pantsel** [comment]
Keep in mind that this is a work in progress. Many things will be changed to the better, including the automated builds.

For now, that is the best solution given the time and the need i had to make and use an image as small as possible.

Automated builds are not a priority for me still, this is something that indeed needs to be improved.
It's  just at the end of my priority stack and i will not dive into it for now.

Right now I'm focusing on decoupling frontend and backend and removing concerns from Konga (like consumers api logic and stuff).

To sum up, if you want to build. you will have to cd frontend && gulp dist first.
If not, you can get the already built image from dockerhub.

Alternatively, you could implement the automated builds with better build-logic yourself, and submit a PR. 

I would be more than grateful since there are a ton of things that need to be done, and i really don't have the time to deal with everything in a short time-span.

**Comment by bakstad** [comment]
See PR #26 for a self building docker image.

**Comment by pantsel** [comment]
Addressed in release https://github.com/pantsel/konga/releases/tag/v0.5.0
